example/test_problem/Hydro/GrackleTest/yt_script/plot_projection.py

```
@@ -0,0 +1,197 @@
+        raise ValueError( 'OPT__OUTPUT_GRACKLE_TEMP must be enabled !!' )
+
+    if not ds.parameters["Opt__Output_GrackleMu"]:
+        raise ValueError( 'OPT__OUTPUT_GRACKLE_MU must be enabled !!' )
+
+    if not ds.parameters["Opt__Output_GrackleTCool"]:
+        raise ValueError( 'OPT__OUTPUT_GRACKLE_TCOOL must be enabled !!' )
+
+
+    def _grackle_T_over_mu( field, data ):
+        return data['grackle_temperature'] / data['grackle_mu']
+    ds.add_field( ('gas', 'grackle_T_over_mu'), function=_grackle_T_over_mu, sampling_type='cell', units='K', display_name='Grackle $T/\mu$'  )
```

Ruff (0.15.5)

[warning] Invalid escape sequence: `\m`

Use a raw string literal

(W605)